src/components/contentBlocks/expandableContentBlock.tsx

```diff
@@ -1,25 +1,78 @@
+import { graphql } from 'gatsby';
 import React, { FC, useState } from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
+
 import { ExpandableContentBlockInformationFragment } from '../../../types/graphql-types';
+import ExpandIcon from '../../icons/icon-expand.svg';
+import * as colors from '../../style/colors';
+import {
+  applyMediaQueryMd,
+  applyMediaQueryLg,
+  contentMargin,
+  contentMaxWidth,
+} from '../../style/dimensions';
 import ContentfulRichText from '../contentfulRichText';
-import { graphql } from 'gatsby';
+import IconButton from '../elements/iconButton';
+import Flex from '../elements/flex';
 
 interface Props {
   data: ExpandableContentBlockInformationFragment;
 }
 
-const Container = styled.div``;
+const Container = styled.div`
+  border-bottom: 1px solid ${colors.Grey300};
+  max-width: ${contentMaxWidth};
+  padding: 24px 0;
+
+  & > * {
+    margin: 0 ${contentMargin.sm};
+  }
+
+  ${applyMediaQueryMd(css`
+    & > * {
+      margin: 0 ${contentMargin.md};
+    }
+  `)}
+
+  ${applyMediaQueryLg(css`
+    & > * {
+      margin: 0 ${contentMargin.lg};
```

Comment:
For me 70px seems to be a lot. The 20px would work perfectly in all resolutions. Also with the current implementation to with of the texts is on 520 pixels (In the screen it's 638). With less margin we could get closer.

Reply:
changed it to reduce the margin to 20 on medium screen and to 0 on large screens. Now it's 660 which is the constant i use everywhere for maxWidth (i have to check though if the margin reduces the text at other places below 660 as well)